assets/models/system/openai-clip-image-text-embeddings-vit-large-patch14-336/description.md

```diff
@@ -0,0 +1,109 @@
+The `CLIP` model was developed by OpenAI researchers to learn about what contributes to robustness in computer vision tasks and to test the ability of models to generalize to arbitrary image classification tasks in a zero-shot manner. The model uses a ViT-L/14 Transformer architecture trained at 336x336 pixel resolution as an image encoder and uses a masked self-attention Transformer as a text encoder. The model was trained on publicly available image-caption data, which was gathered in a mostly non-interventionist manner. The model is intended as a research output for research communities, and the primary intended users of these models are AI researchers. The model has been evaluated on a wide range of benchmarks across a variety of computer vision datasets, but it currently struggles with respect to certain tasks such as fine-grained classification and counting objects. The model also poses issues with regards to fairness and bias, and the specific biases it exhibits can depend significantly on class design and the choices one makes for categories to include and exclude.
```

Comment:
please align description as this [doc](https://microsoftapc-my.sharepoint.com/:w:/r/personal/ayushmishra_microsoft_com/_layouts/15/doc2.aspx?sourcedoc=%7B2E13DB38-A1A9-4148-B1DC-AC14A76E1F72%7D&file=System%20Registry%20model%20card%20alignment.docx&nav=eyJjIjo0MTgzNzU3OX0&action=default&mobileredirect=true&share=IQE42xMuqaFIQbHcrBSnbh9yAamNOJxrsZ9DvamUluUGp40&wdOrigin=TEAMS-ELECTRON.p2p_ns.bim&wdExp=TEAMS-CONTROL&wdhostclicktime=1703143808964&web=1).

Also you can refer to this [PR] (https://github.com/Azure/azureml-assets/pull/2010)